api/lib/mno_enterprise/concerns/controllers/jpi/v1/impac/alerts_controller.rb

```diff
@@ -22,10 +22,10 @@ def create
 
     authorize! :manage_alert, kpi_alert
 
-    if (@alert = current_user.alerts.create(kpi_alert.attributes))
+    if (@alert = kpi_alert.save(kpi_alert.attributes))
```

Comment:
That would be perfect but unfortunately this introduces a backward incompatibility with mno-hub: we have to make sure that it doesn't break if we are to release a frontend (mnoenterprise + impac-angular) without releasing mnohub...

@x4d3 happy to have your opinion on that one

```ruby
kpi_alert.save(kpi_alert.attributes)
# TODO: remove following once all mnohub are migrated to version > ?
if kpi_alert.attributes[:recipients].present? && kpi_alert.recipients.empty?
  recipients = kpi_alert.attributes.delete(:recipients)
  recipients.each do |recipient_hash|
    # mno_user similar to what is done in mnohub
    mno_user(recipient_hash)&.alerts.create(kpi_alert.attributes)
  end
end
```